### PR TITLE
Readds the release sequence for cryogenics and buffs it

### LINKED ISSUE #2844: Cryo is weird for occupant
##### Revision that this bug was encountered on (REQUIRED!! Type 'show-server-revision' ingame to find out - note that 'current revision' isn't helpful!): #####
2014-02-19
a92cce91639314c181ce71f9f5a7ee35567f6d98

##### Which server were you playing on? (REQUIRED!! e.g. TG-Sibyl1, TG-Sibyl2, Badgerstation, Facepunch, Pinkstation, /VG/... etc.) #####
Private server

###### What did you expect to happen? ######
To have a clear and understandable way of leaving cryo, like moving out or resisting.

###### What happened instead? ######
You will convert to a normal sleep if fully healed.
Then you'll wake up, while still frozen.
When you move out of the cryo or resist to get out after waking up, you take burn damage from the cold.
If you're not fully healed there is no way to get out by yourself.

###### Why is this bad/What are the consequences? ######
This is weird.

##### Steps to reproduce the problem (remember to be specific! Those little details can save us hours of work!): #####
Get in cryo with full health.
Turn it on.
Wake up automatically.

Get in cryo with 99 health.
Turn it on.
Frozen forever.

##### Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): #####
#2488

##### Anything else? #####
@Razharas

### PR DESCRIPTION
**Pull request by ghost**
Readds the release sequence for cryogenics and makes it 1 minute instead of 2 minutes, considering handcuff release time was nerfed the same.

Fixes https://github.com/tgstation/-tg-station/issues/2844

### CODE REVIEW

**Comment by dannno** [comment]
FIIIIIIIIIIIIIIIINALLY :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by TZK13** [comment]
It's mind-boggling how long you can be left in a cryo cell with people just passing by sometimes, a much-needed quality of spesslife improvement. :+1:

**Comment by Iamgoofball** [comment]
I also made cryotubes spit people at full health out automatically in the Addiction PR for goonchem

:+1:

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -81,12 +81,26 @@
 	open_machine()
 
 /obj/machinery/atmospherics/unary/cryo_cell/container_resist()
-	if(stat & DEAD)
-		return
-	sleep(usr.stat * 1200)
 	open_machine()
 	return
 
+/obj/machinery/atmospherics/unary/cryo_cell/verb/move_eject()
```
Comment (tkdrg) [inline code/game/machinery/cryo.dm]: Why object veeeerbs
Can't you tie this behavior into something like container_resist() or relay_move() instead

Comment (paprka) [inline code/game/machinery/cryo.dm]: no

Comment (tkdrg) [inline code/game/machinery/cryo.dm]: Being dead blocks this too. I don't see your point?

Comment (paprka) [inline code/game/machinery/cryo.dm]: I mean unconscious, sorry

**Comment by Razharas** [comment]
I think we all can agree that object verbs are bad
Maybe there can be some cases that make resist work while unconcious or some other ways

**Comment by paprka** [comment]
I don't care if object verbs are bad, stalling this feature any longer which you removed is a worse idea

If you want to improve it, go for it, but this was an undocumented removal and really, really shitty.

**Comment by Kromgar** [comment]
Undocumented like your changes to the gyrojet?

**Comment by paprka** [comment]
Wasn't the gyrojet nerf coupled with like 500000 other gun changes?

Take your bitter bullshit elsewhere you cancer.

**Comment by Kromgar** [comment]
But mah hypocrisy

**Comment by Cheridan** [comment]
Can anyone propose a solution that that would work that isn't Object Verbs? Being unconscious blocks resisting and some others, so that's an issue. A verb is probably the best way here.

**Comment by paprka** [comment]
Trust me I tried to find a workaround so if you moved when unconscious it'd still activate the release sequence but I couldn't get it to work. I think the same thing that makes you a ghost when you're dead and you move might work, but it wouldn't function fro me.

**Comment by Steelpoint** [comment]
Just so we know but if your in positive health inside a Cyro tube, your fully concious and you can freely leave the tube at any point. Meaning there's no need to have a automatic ejection system.

**Comment by paprka** [comment]
Yeah idk why goof added that but it's not my feature so I don't really care so much

**Comment by Iamgoofball** [comment]
i added it because cryo was getting buffed and so people would be using cryo more, increasing the likelyhood of "stuff 'em in cryo and go take a shit" and then they never come back and pull you out of cryo, resulting in you being stuck in cryo for the rest of the round

so i made it spit you out if you are at full HP because of 2 reasons:
1. getting stuck in cryo was boring as shit
2. if you're at full HP, if you're still in the cell it's gonna keep pumping cryox into you, wasting the remaining cryox on a fully healthy guy when it's needed for the next in crit guy nearby.

**Comment by paprka** [comment]
Yeah I'm fine with it then but resisting out when unconscious should be a thing too imo

**Comment by Iamgoofball** [comment]
ye, i approve of that being added too for if you really don't have time to wait on cryo

**Comment by paprka** [comment]
R E V I E W
E
V
I
E
W

**Comment by Miauw62** [comment]
ayy lmao object verbs are shit but apparently the only way to get people to fix shit.

**Pull request merged**